src/services/InputController.java

```diff
@@ -0,0 +1,159 @@
+package services;
+
+import custom_exceptions.*;
+import main.Faculty;
+import java.util.Scanner;
+
+public class InputController {
+    private static Scanner scanner = new Scanner(System.in);
+    private static int input;
+
+    public static int checkForWrongInput(String nameOfSpecificGroup){
+        while(true){
+            try {
+                input = Integer.parseInt(scanner.nextLine());
+                if(nameOfSpecificGroup.equals("marks") && (input<0 || input>10)){
```

Comment:
Consider renaming appropriately  namesOfSpecificGroup since you are checking for the grade range which doesn't depend on the name of a group